src/pages/twap/TWAPTable.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { Tabs, type TabItem } from '@/components/Tabs';
+import { ActiveTWAPTable, ActiveTWAPTableColumnKey } from './ActiveTWAPTable';
+
+const tabItems: TabItem<string>[] = [
+  {
+    value: 'Active',
```

Comment:
make values an enum

Reply:
or at least make it a type

type TWAPTableTabs = 'Active' | 'OrderHistory' | 'Fills';

Reply:
and use with the useState<TWAPTableTabs>('Active')